### PR TITLE
feat: Adds the list messages action

### PR DESCRIPTION
**Pull request by Blond11516**
Implements the action to list messages.

Part of https://github.com/ZeLarpMaster/Mobius/issues/29.

### CODE REVIEW

File: lib/mobius/actions/message.ex [hunk 1]
```diff
@@ -73,4 +75,16 @@ defmodule Mobius.Actions.Message do
         Rest.Message.send_message(Bot.get_client!(), channel_id, body)
     end
   end
+
+  def list_messages(channel_id, params) do
+    # TODO: Validate permissions
+    # TODO: Make sure the cannel exists (requires a cache)
+
+    validators = [integer_range_validator(:limit, 1, 100)]
```
Comment (ZeLarpMaster) [inline lib/mobius/actions/message.ex]: Any reason for not validating the other params?

Comment (Blond11516) [reply lib/mobius/actions/message.ex]: Nope, just forgot to do it :stuck_out_tongue:

Comment (Blond11516) [reply lib/mobius/actions/message.ex]: Done

File: lib/mobius/validations/action_validations.ex [hunk 2]
```diff
@@ -6,34 +6,56 @@ defmodule Mobius.Validations.ActionValidations do
...
+  @spec integer_validator(Map.key()) :: validator()
+  def integer_validator(key) do
+    fn
+      %{^key => val} when is_integer(val) -> :ok
+      %{^key => val} -> {:error, "Expected #{key} to be an integer, got #{inspect(val)}"}
+      _ -> :ok
+    end
+  end
+
+  def snowkflake_validator(key) do
```
Comment (ZeLarpMaster) [inline lib/mobius/validations/action_validations.ex]: `snowkflake` --> `snowflake`

**Review by ZeLarpMaster — APPROVED** [review]

**Pull request merged**